jstz_api/src/console.rs

```diff
@@ -234,6 +297,7 @@ impl Console {
         context: &mut Context<'_>,
     ) -> JsResult<()> {
         LogMessage::Warn(formatter(data, context)?).log(rt, self);
+        LogMessage::Warn(formatter(data, context)?).json_log(rt, self);
         Ok(())
     }
```

Comment:
Instead of doing both of logs, I think we should have a feature flag to determine whether we want 'pretty' logs or 'json' logs